client_it_test.go

```diff
@@ -1043,6 +1044,62 @@ func clientStartShutdownWithNilContextTest(t *testing.T) {
 	it.Must(client.Shutdown(nil))
 }
 
+func clusterConnectionToMultipleAddrs(t *testing.T) {
```

Comment:
Could you add a `Test` suffix to this function's name?

Reply:
Sure thanks https://github.com/hazelcast/hazelcast-go-client/pull/885/commits/b6c392dce32ad6f9628881f5492531354940969d